src/hooks/useKeyboardShortcut.js

```diff
@@ -10,28 +10,22 @@ import KeyboardShortcut from '../libs/KeyboardShortcut';
  */
 export default function useKeyboardShortcut(shortcut, callback, config = {}) {
     const {captureOnInputs = true, shouldBubble = false, priority = 0, shouldPreventDefault = true, excludedNodes = [], isActive = true} = config;
-
-    const subscription = useRef(null);
-    const subscribe = useCallback(
-        () =>
-            KeyboardShortcut.subscribe(
+    const handleEvent = useEffectEvent(() => callback());
```

Comment:
Is it possible that `useEffectEvent` is able to fix the references inside the function `callback` so they see the most up to date state variables/props? This sounds highly unlikely to me. Is this how it is supposed to be used?

We can test if the `callback` during execution is looking at the latest values or if stayed frozen in time.